### ISSUE TITLE
Vanilla docs aren’t searchable except for their titles

### ISSUE THREAD

**Issue by matthewpaulthomas** [canonical-web-and-design/vanillaframework.io#181]
This just happened to me:

1\. Try to remember the name of the Vanilla text field that has a Copy button.
2\. Go to the docs site.
3\. Click in the “Search components” field and type “copy”.

What happens: No search results. In particular, [“Code snippet”](https://docs.vanillaframework.io/en/patterns/code-snippet) is not returned as a search result, despite its page using the word “copy” three times, including in its first sentence.

What should happen: “Code snippet” is returned as a search result.

This is because the search is currently just on page titles, rather than pages as a whole. The title search is useful, and could be retained, but it is not a substitute for a full search.

This is the sequel to #105.

**Comment by steverydz** [comment]
I too have come across this. For me the search isn't working at all. There is an error in the console:

```
Uncaught TypeError: Cannot read property 'classList' of null at filterDocs (scripts.js:38)
at HTMLInputElement.oninput (docs.vanillaframework.io/:58)
```

**Comment by kwm14** [comment]
Thanks, @steverydz I spotted this on Friday too. I've reported this [bug](https://github.com/canonical-web-and-design/vanilla-framework/issues/2407) and will be fixed this iteration 👍

**Comment by bartaz** [comment]
Fixed by #2822
